prebuilt/system/etc/mixer_paths.xml

```diff
@@ -1614,7 +1614,7 @@
         <ctl name="RDAC3 MUX" value="DEM2" />
         <ctl name="DAC1 Switch" value="1" />
         <ctl name="EAR PA Gain" value="POS_6_DB" />
-        <ctl name="RX1 Digital Volume" value="84" />
+        <ctl name="RX1 Digital Volume" value="75" />
```

Comment:
I really don't want to lower incall earpiece volume, sometimes loud volume is needed, e.g. noisy environment or low mic volume by other party.
And it's easy enough to lower volume by your self when needed.